start-local.sh

```diff
@@ -374,18 +376,21 @@ if [ "$need_wait_for_kibana" = true ]; then
 fi
 
 # Success
-echo "Congrats, Elasticsearch and Kibana successfully installed!"
 echo
-echo "Open the browser at http://localhost:5601"
+echo "🎉 Congrats, Elasticsearch and Kibana are successfully installed and running!"
+echo
+echo "🌐 Access Kibana at http://localhost:5601"
```

Comment:
Maybe we want to print the elasticsearch URL too or is that overkill?

It might simplify client instantiations to have the ES endpoint printed to the terminal along with with API key

Reply:
I think printing the ES endpoint will also help to mitigate #4, good idea. I also like the emojis! Thanks.